pkg/controller/infrastructure/infraflow/reconcile.go

```diff
@@ -522,22 +568,46 @@ func (c *FlowContext) ensureZones(ctx context.Context) error {
 		tagsPrivate[TagKeyRolePrivateELB] = TagValueUse
 		desired = append(desired,
 			&awsclient.Subnet{
-				Tags:             tagsWorkers,
-				VpcId:            c.state.Get(IdentifierVPC),
-				CidrBlock:        zone.Workers,
-				AvailabilityZone: zone.Name,
+				Tags:      tagsWorkers,
+				VpcId:     c.state.Get(IdentifierVPC),
+				CidrBlock: zone.Workers,
+				Ipv6CidrBlocks: func() []string {
+					if subnetCIDR1 != "" {
+						return []string{subnetCIDR1}
+					} else {
+						return nil
+					}
+				}(),
+				AvailabilityZone:            zone.Name,
+				AssignIpv6AddressOnCreation: pointer.Bool(false),
```

Comment:
What does this mean? What would happen if we would set this to `true`? Would that automatically assign a `/64` block to the subnet so that we would not have to do the IP address management?

Reply:
No, if we set it to `true`,  ipv6 addresses would automatically be assigned to instances.